src/Raven.Server/Documents/Handlers/Admin/RachisAdminHandler.cs

```diff
@@ -453,6 +454,11 @@ public async Task AddNode()
 
                     await ServerStore.AddNodeToClusterAsync(nodeUrl, nodeTag, validateNotInTopology: true, asWatcher: watcher ?? false);
 
+                    if (LoggingSource.AuditLog.IsInfoEnabled)
+                        LogAuditFor("Server", "PUT", $"ADD Node {nodeTag} to cluster. Term: {ServerStore.Engine.CurrentTerm}.");
```

Comment:
```suggestion
                        LogAuditFor("Server", "ADD", $"Node {nodeTag} to cluster. Term: {ServerStore.Engine.CurrentTerm}.");
```

Reply:
This way the log entry will be `ADD Node ...` instead of `PUT ADD Node ...`

Reply:
Done